lms/templates/admin/registration.html.jinja2

```diff
@@ -11,7 +11,7 @@
     <fieldset class="box">
     <legend class="label has-text-centered">Registration application instances</legend>
         <div class="block has-text-right">
-            <a class="button is-primary" href="{{ request.route_url("admin.registration.new.instance", id_=registration.id) }}">New instance</a>
+            <a class="button is-primary" href="{{ request.route_url("admin.registration.new.instance", id_=registration.id) }}">Add instance</a>
```

Comment:
At this point we are adding a new instance to this registration. It could be a new one or upgrading an existing one.

Making the language a bit clearer.